cves/CVE-2018-11784.yml

```diff
@@ -120,10 +126,12 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
-  contest:
+  answer: |
+    Looking at the emails and the initial dates, Jun 5th 2018 seems to be the 
+    closest I'll get to the day this bug was actually discovered. 
+  date: 2018-06-05
```

Comment:
This date is a lot sooner than the date you said that the bug was reported to the team

Reply:
This was an err on my part, the reported date should've read the same as the discovered date. It was the earliest sate I found